timeseries/src/autogluon/timeseries/evaluator.py

```diff
@@ -129,101 +121,45 @@ def coefficient(self) -> int:
     def higher_is_better(self) -> bool:
         return self.coefficient > 0
 
-    def _safemean(self, data: Any):
-        data_filled = np.nan_to_num(data, neginf=np.nan, posinf=np.nan, nan=np.nan)
```

Comment:
Old code: If a time series contains a single time step with NaN metric, the entire time series is ignored when computing the metric.

New code: `mean` in Pandas ignores the timesteps with `NaN` metrics and computes the average over other valid timesteps.

Reply:
seems fine to me, doubt we need to communicate with users, it isn't too big of a change. If you think it is important, feel free to say so.